.circleci/config.yml

```diff
@@ -2,11 +2,12 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/golang:1.17
+      - image: cimg/go:1.21
```

Comment:
As per https://hub.docker.com/r/cimg/go `circleci/golang` was deprecated in favor of `cimg/go`